sidebarsDocs.js

```diff
@@ -1200,5 +1200,21 @@ module.exports = {
       items: ['use-the-network/devices/development/heltec/wifi-lora-32-v2/arduino'],
       collapsed: false,
     },
-  ]
+  ],
+  abeeway: [
```

Comment:
This name is already being used by another manufacture, make this `origo` instead.